slm_porosity/heat_transfer.f90

```diff
@@ -718,7 +716,7 @@ SUBROUTINE check_user_input
     IF (.NOT.laser_inside_domain) &
       STOP '--- The initial laser position is out of domain ---'
     IF (dim.EQ.3 .AND. half_domain .AND. xyzlimits(1,2).NE.0.0_pr) &
-      STOP '--- half domain = true, the initial laser position does not located on the boundary ---'
+      STOP '--- The initial laser position is not on the plane y=0 ---'
```

Comment:
Without a comment about halfdomain it source of the error is non understandable. y in [-5 0] and y in [0 5] are differs in this condition. But there nothing about it.

Reply:
You are right. I tried to make comment more laconic.